pahfit/base.py

```diff
@@ -320,11 +321,13 @@ def plot(self, axs, x, y, yerr, model, scalefac_resid=2):
         # now plot the dust and gas lines
         for cmodel in model:
             if isinstance(cmodel, Gaussian1D):
+                # ax.plot(x, (cont_y + cmodel(x)) * ext_model / x, "#DC267F", alpha=0.5)
```

Comment:
Why is this added?

Reply:
Same as the above comment. Will be removed.